icons/square/48/mailspring.svg

```diff
@@ -0,0 +1,17 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="48" viewBox="0 0 48 48.000001" width="48" version="1.1">
```

Comment:
* `height` and `width` is not needed
* the `viewBox` height dimension should only contain integers